### ISSUE TITLE
Asp.net core 2 + Angular 4+ SPA SEO issues 

### ISSUE THREAD

**Issue by marvelTracker** [MarkPieszak/aspnetcore-angular2-universal#575]
Hi There,

I've used the ASP.net core 2 + Angular 4+ SPA template for one of my projects. It worked really well and I deployed it on Azure.

However, when I tried to index with google bot, it didn't correctly identify my content. I've attached the screen shot how the google bot renders the page and it didn't render the html in angular app

![11](https://user-images.githubusercontent.com/1152084/37744363-beabf836-2dc2-11e8-8267-c5fe699007da.PNG)
![22](https://user-images.githubusercontent.com/1152084/37744364-bede21da-2dc2-11e8-80bd-3088489d0a01.PNG)

I really appreciate your help on this.

Thanks,

**Comment by chrisvfabio** [comment]
When you right click and View Page Source, does it render the Angular app? Maybe there is an error in NodeServices when it renders the app.server.module. 

Does it work locally?

**Comment by MarkPieszak** [comment]
Yeah double check if it works locally, it sounds like there's an error during SSR so it's returning just the regular client-side application.

Or did you shut off server-side rendering? 
The _index.cshtml file should have aspnet tags in there where it's outputting the result of the serialized application.

**Comment by marvelTracker** [comment]
Hi Guys, 

Thanks for the feedback. I deployed sample to this location http://dronescomparer.com/fetch-data 
This is the page I'm refereeing. It worked locally without issues.

**Comment by MarkPieszak** [comment]
Make sure Azure is set to use Node 8.x+ as well.
